packages/fast-tooling-react/src/form/form.tsx

```diff
@@ -461,4 +553,4 @@ class Form extends React.Component<
 }
 
 export { FormChildOptionItem, FormClassNameContract };
-export default manageJss(styles)(Form);
+export default DragDropContext(HTML5Backend)(manageJss(styles)(Form));
```

Comment:
Were we creating a context before?

Reply:
Fixed! Thanks, that was a bad merge I missed.